two-photon/rip.py

```diff
@@ -35,10 +35,10 @@ def determine_ripper(data_dir, ripper_dir):
     version = root.attrib['version']
 
     # Prairie View versions are given in the form A.B.C.D.
-    match = re.match(r'^(?P<majmin>\d+\.\d+)\.\d+\.\d+$', version)
+    match = re.match(r'^d+\.\d+\.\d+\.\d+$', version)
```

Comment:
At this point, we don't likely need a regex, since we are going to use the whole version.  We can likely just drop this.

Reply:
Sounds good to me! I'll update that.